### DISCUSSION TITLE
Ability to Enable/Disable Listings for a Forem

### DISCUSSION THREAD

**Discussion by nickytonline** [forem/forem#16429] [Feature Requests]
Currently, there is no way to disable listings for a Forem instance that I'm aware of. For my forem, vscodetips.com, listings make no sense for the time being, and as well, I don't have Stripe set up if members try to purchase credits.

It would be nice to be to opt-in or opt-out of this feature.

**Comment by citizen428** [comment]
This is already a WIP, @jeremyf is working on it.
